### DISCUSSION TITLE
monorepo

### DISCUSSION THREAD

**Discussion by karlAlnebratt** [rafgraph/rollpkg#1] [General]
Hi do you have an example on how to set hit up in a monorepo? 

ex:
```
- packages
  - code1
    - package.json
  - code2
    - package.json
package.json
tsconfig.json   
```

**Comment by rafgraph** [comment]
Unfortunately I don’t. I’d try adding it to each package’s `package.json`

**Comment by rafgraph** [comment]
I think you should be able to install `rollpkg` once at the top level, and only have the `rollpkg build/watch` command in each package’s `package.json`, but I haven’t tested this out.
